test/config.yaml

```diff
@@ -3,4 +3,4 @@ genome_file: test_files/genome.fa.gz
 gtf_file: test_files/gene_annotations.gtf.gz
 mirna_file: test_files/mirna_annotations.gff3
 map_chr_file: test_files/ucsc_to_ensembl.txt
-mir_list: ["miRNA", "miRNA_primary_transcript"]
\ No newline at end of file
+mir_list: ["miRNA"]
```

Comment:
Why does this not work anymore with `"miRNA_primary_transcript"`?

Reply:
This does not work anymore with "miRNA_primary_transcript" as now the annotation file does only contain either "miRNA" or "miRNA_primary_transcript" and when merging the tables it looks for them. I thought that for this PR I would delete the "miRNA_primary_transcript" and two issues ahead (#69) when extending the BAM file with the extended primiRs, I would add it again and uncomment the pri-mirs table rule.

Reply:
Couldn't you just add the new script/rule without changing/removing any other rules? Removing something isn't really part of this issue. We can clean up later.